### PR TITLE
[Port]New 25TC bundle, Supermatter delaminator

### PR DESCRIPTION
**Pull request by Ghilker**
### This PR is ported from github.com/yogstation13/Yogstation/pull/9037
Adds the supermatter delaminator, a 25TC that will guarantee a delamination of your local supermatter crystal. Comes in a syndie box kit, complete with instructions and extraneous equipment.

Main difference is that the delamination can be stopped by surrounding the SM with Hypernoblium at over 10 moles per tile

The original PR has been slighly changed to adjust on TG. There may be artifacts from Yog. I fixed a bunch of things but there will be many  that i missed.

## Why It's Good For The Game
From the original PR
The Supermatter can now be seeded with an antinoblium shard and a corruptor, corrupting the support infrastructure that regulates it. The effects of this are not immediately felt, but when they do the entire station will feel it. The supermatter's supporting platform, as well as the radio on it and the power shield that regulate it are corrupted and slowly degrade. This support integrity starts at 100 and decreases by 1 every time the supermatter goes BRRRRRRR. As the integrity goes lower, the SM becomes more deadly, releasing more gas (scales with support integrity), increasing its power all by itself (scales), releasing shock arcs (at stability <70) to complicate repair and stabilization efforts and even radballs when stability dips below 40, complete with starting its own fires. When stability hits 10, it starts emitting massive EMP waves to ensure that any active cooling that has managed to keep it stable up until this point stops functioning, whereupon it will go on its merry way BRRRRRRRing until full delamination.

This whole process is balanced by the fact that the degradation of the support infrastructure takes about 5~8 minutes before it hits 0 and starts delaminating, granting ample time for engineers to prepare emergency cooling and for the shuttle to be called. The size of the SM explosion is also increased, though this can be tweaked if desired.

## Changelog
:cl:
add: supermatter delaminator, a 25TC bundle to delaminate the SM and cause havoc
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
this reinforces destroying the entire station as an antag tbf

**Comment by MrDoomBringer** [comment]
> granting ample time for ... the shuttle to be called.

homie you sure this will actually be fun for anyone?

**Comment by MrDoomBringer** [comment]
this feels kinda meh compared to something of a similar cost like romerol

**Comment by Ghilker** [comment]
i wanted to lock this behind glorious death objective, but there actually isn't a way to do it (like on yog side) that's why is 25 tc bundle

**Comment by Qustinnus** [comment]
that won't make it more fun.

**Comment by Misdoubtful** [comment]
Considering that this is 25TC, why would someone use this over something like, shooting the SM with a gun? Doing so is just easier, cheaper, and faster.

**Comment by Ghilker** [comment]
> Considering that this is 25TC, why would someone use this over something like, shooting the SM with a gun? Doing so is just easier, cheaper, and faster.

I don't know, i think is funny to see everyone panic when the SM start saying weird stuff and the numbers are randomized that much that it doesn't make sense

**Comment by brotherbeyondd** [comment]
why do any of that when funny pistol go brrr

**Comment by itseasytosee** [comment]
Blowing shit up with bombs when you want to die is a lot more fun then gimping the station's power network.

**Comment by Jared-Fogle** [comment]
I don't see how a 25 TC "call the shuttle" item is not interesting for anyone. We already have something like this in the form of Romerol in that it nearly always guarantees the shuttle is called, but that's a lot more interactive for everyone.

**Comment by AutisticFroggy** [comment]
Original PR author here. You actually _can_ stop the SM from blowing up when seeded with an antinoblium shard, it's just _really hard_ and presents a difficult but solvable engineering problem. I understand that teegee engineers are less robust than yogstation engineers, so you may want to nerf the hotspot min arg and slow down the delam damage by 50% to give them more time.

**Comment by Ghilker** [comment]
> I don't see how a 25 TC "call the shuttle" item is interesting for anyone. We already have something like this in the form of Romerol in that it nearly always guarantees the shuttle is called, but that's a lot more interactive for everyone.

this is interesting for the Engineering department as a whole. Also nobody is forcing you to take this over romerol, this is another of the many ways to delaminate the SM 

> why do any of that when funny pistol go brrr

because that's lame

File: code/game/objects/items/theft_tools.dm [hunk 1]
```diff
@@ -233,7 +233,7 @@
 	. = ..()
 	usesLeft = rand(2, 4)
 
-/obj/item/hemostat/supermatter
+/obj/item/supermatter
```
[the remark this answers is not in the document]
Comment (Ghilker) [reply code/game/objects/items/theft_tools.dm:236]: f

**Comment by itseasytosee** [comment]
> because that's lame

Is it lame because theirs no counterplay or because it doesn't blast warnings over comms?

**Comment by itseasytosee** [comment]
I don't know if this did some sort of cool delam with special effects or whatever id be on board with it but right now its kinda useless.

**Comment by optimumtact** [comment]
This is total cringe, and removes skill

**Comment by ATH1909** [comment]
> > Considering that this is 25TC, why would someone use this over something like, shooting the SM with a gun? Doing so is just easier, cheaper, and faster.
> 
> I don't know, i think is funny to see everyone panic when the SM start saying weird stuff and the numbers are randomized that much that it doesn't make sense

Okay, then just sabotage the SM normally without spending 25 TC on doing so?

Also, don't we already have the "steal an SM sliver" objective for this kind of thing? If that isn't up to snuff, why not buff that instead of introducing a 25 TC item that no one will purchase ever except as golden syndicate balloon-like item?

**Comment by wesoda25** [comment]
Bro it's interesting for the engineering department and an inevitable round ender for everyone else. Shouldn't we encourage people to understand the SM mechanics and do interesting delams? Just sort of doesn't seem very fair for 80% of the station.

**Comment by LemonInTheDark** [comment]
Delaming the sm should rely more on knowledge of the thing rather then buying an item. The closest we have to something like that is the scalpel, and it's not remotely comparable to this.

I don't like the idea of something like this, and want any massive theatrical stuff like this to be accessible to most anyone, see singulo or tesla delams. I haven't reviewed the code either, but from a glance large emp waves sound very worrying.

**Comment by Ghilker** [comment]
Aight

**Pull request closed without merging**